tox.ini

```diff
@@ -30,7 +30,10 @@ deps =
     cms34: https://github.com/divio/django-cms/archive/release/3.4.x.zip
     cms35: https://github.com/divio/django-cms/archive/release/3.5.x.zip
     cms36: https://github.com/divio/django-cms/archive/release/3.6.x.zip
-    cms37: https://github.com/divio/django-cms/archive/release/3.7.x.zip
+    py27-django111-cms37: django-cms==3.7.1
```

Comment:
This is not required anymore, but maybe it comes from an update from and older develop branch

Reply:
you can just revert the changes in `tox.ini`. It used to be required due to a bug in django-cms that has been fixed, so it's safe to use the `3.7.x` branch for all the python versions

Reply:
Okay, I will replace 
```
    py27-django111-cms37: django-cms==3.7.1
    py35-django111-cms37: cms37: https://github.com/divio/django-cms/archive/release/3.7.x.zip
    py36-django111-cms37: cms37: https://github.com/divio/django-cms/archive/release/3.7.x.zip
    py37-django111-cms37: cms37: https://github.com/divio/django-cms/archive/release/3.7.x.zip
```
by
```
    cms37: https://github.com/divio/django-cms/archive/release/3.7.x.zip
```